internal/jobs_test.go

```diff
@@ -0,0 +1,10 @@
+package internal
+
+import (
+	"testing"
+)
+
+func TestCreateJob(t *testing.T) {
```

Comment:
This test won't be running (and wasn't running in your example) as part of integration test. In order to be run it should be named `TestAccCreateJob`